content/faq/what-file-types-are-scanned/index.md

```diff
@@ -0,0 +1,34 @@
+---
+title: "What file types are scanned?"
```

Comment:
The title suggests it's comprehensive of all file types we scan, but the content appears limited to just the archive file types.

Perhaps change the title to "What archive file types are scanned?"

Reply:
We are saying `BluBracket scans for code risks in all text encoded files`. Do you want it to be highlighted?

Reply:
Yeah, it definitely got lost.